helm/idol-nifi/resources/flow-basic-idol.json

```diff
@@ -1 +1 @@
ld":{"displayName":"Table Entity Field","identifiesControllerService":false,"name":"TableEntityField","sensitive":false},"MaxSearchHeaderRow":{"displayName":"Table Max Search Header Row","identifiesControllerService":false,"name":"MaxSearchHeaderRow","sensitive":false},"CaseNormalizationBehavior":{"displayName":"Case Normalization Behavior","identifiesControllerService":false,"name":"CaseNormalizationBehavior","sensitive":false},"PostProcessThreshold":{"displayName":"Post Process Threshold","identifiesControllerService":false,"name":"PostProcessThreshold","sensitive":false},"PostProcessingThreshold":{"displayName":"PostProcessingThreshold","identifiesControllerService":false,"name":"PostProcessingThreshold","sensitive":false},"TangibleCharacters":{"displayName":"Tangible Characters","identifiesControllerService":false,"name":"TangibleCharacters","sensitive":false},"RedactedOutput":{"displayName":"Redacted Output","identifiesControllerService":false,"name":"RedactedOutput","sensitive":false},"TableCellValuePath":{"displayName":"Table Cell Value Path","identifiesControllerService":false,"name":"TableCellValuePath","sensitive":false},"ResourceFiles":{"displayName":"Resource Files","identifiesControllerService":false,"name":"ResourceFiles","sensitive":false},"MatchCase":{"displayName":"Match Case","identifiesControllerService":false,"name":"MatchCase","sensitive":false},"RedactionReplacementCharacter":{"displayName":"Redaction Replacement Character","identifiesControllerService":false,"name":"RedactionReplacementCharacter","sensitive":false},"PostProcessingTask":{"displayName":"Post Processing Tasks","identifiesControllerService":false,"name":"PostProcessingTask","sensitive":false},"AllowMultipleResults":{"displayName":"Allow Multiple Results","identifiesControllerService":false,"name":"AllowMultipleResults","sensitive":false},"[PIIValidation]Entities":{"displayName":"[PIIValidation]Entities","identifiesControllerService":false,"name":"[PIIValidation]Entities","sensitive":false}},"retriedRelationships":[],"retryCount":10,"runDurationMillis":0,"scheduledState":"ENABLED","schedulingPeriod":"0 sec","schedulingStrategy":"TIMER_DRIVEN","style":{},"type":"idol.nifi.processor.Eduction","yieldDuration":"1 sec"}],"remoteProcessGroups":[],"variables":{}},"flowEncodingVersion":"1.0","parameterContexts":{"IDOL NiFi Flow Parameters":{"componentType":"PARAMETER_CONTEXT","inheritedParameterContexts":[],"name":"IDOL NiFi Flow Parameters","parameters":[{"description":"","name":"IndexHost","provided":false,"sensitive":false,"value":"idol-index-service"},{"description":"","name":"LicenseServerHost","provided":false,"sensitive":false,"value":"idol-licenseserver"},{"description":"","name":"IndexACIPort","provided":false,"sensitive":false,"value":"9070"},{"description":"","name":"LicenseServerACIPort","provided":false,"sensitive":false,"value":"20000"}]}},"parameterProviders":{},"snapshotMetadata":{"author":"anonymous","comments":"","timestamp":1699008213640,"version":2}}
```

Comment:
In the parameterized file, there's an "IdolLicenseServiceImpl" with
```json
"properties": {
    "LicenseServerHost": "#{LicenseServerHost}",
    "LicenseServerPort": "#{LicenseServerACIPort}"
}
```
but also a "FileSystem Connector Group IdolLicenseService" with
```json
"properties": {
    "LicenseServerHost": "${license.server.hostname}",
    "LicenseServerPort": "20000"
}
```
Guessing that last port should also be parameterized?

Reply:
Just noticed the format of the hostname is different too.

Reply:
[41c794d](https://github.com/opentext-idol/idol-containers-toolkit/pull/12/commits/41c794d8bdd5e7cad9638655c7111306f1baf547)

Reply:
👍 

The FileSystem Connector Group has a now-redundant `license.server.hostname` variable; don't know whether this shows up in any way.

Reply:
[25ce39a](https://github.com/opentext-idol/idol-containers-toolkit/pull/12/commits/25ce39ace958b3b7020bbfc0ba36accb5a79cb20)